pkg/util/k8sutil/crd.go

```diff
@@ -51,19 +51,18 @@ func listClustersURI(ns string) string {
 	return fmt.Sprintf("/apis/%s/namespaces/%s/%s", api.SchemeGroupVersion.String(), ns, api.CRDResourcePlural)
 }
 
-func CreateCRD(clientset apiextensionsclient.Interface) error {
+func CreateCRD(clientset apiextensionsclient.Interface, crdName, rkind, rplural string) error {
 	crd := &apiextensionsv1beta1.CustomResourceDefinition{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: api.CRDName,
+			Name: crdName,
 		},
 		Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
 			Group:   api.SchemeGroupVersion.Group,
 			Version: api.SchemeGroupVersion.Version,
 			Scope:   apiextensionsv1beta1.NamespaceScoped,
 			Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
-				Plural:     api.CRDResourcePlural,
-				Kind:       api.CRDResourceKind,
-				ShortNames: []string{"etcd"},
```

Comment:
This removes *ShortNames*. It shouldn't, right?

Reply:
I am unsure on how short name is being used. do you know if is needed? maybe i can just add another parameter call `shortName`.

Reply:
> maybe i can just add another parameter call shortName.

Right.